packages/dataviews/src/field-types/date.ts

```diff
@@ -52,5 +63,28 @@ export default {
 		return getFormattedDate( value );
 	},
 	enableSorting: true,
-	filterBy: false,
+	filterBy: {
+		defaultOperators: [
+			OPERATOR_ON,
+			OPERATOR_NOT_ON,
+			OPERATOR_BEFORE,
+			OPERATOR_AFTER,
+			OPERATOR_BEFORE_INC,
+			OPERATOR_AFTER_INC,
+			OPERATOR_IN_THE_PAST,
+			OPERATOR_OVER,
+			OPERATOR_BETWEEN,
```

Comment:
I don't mind reusing `between` for date ranges, but the display of date ranges in the filter chip is not clear:

<img width="338" height="50" alt="Screenshot 2025-07-28 at 16 34 39" src="https://github.com/user-attachments/assets/1668a8e8-3a70-44ac-accc-39ac8e599500" />

Throwing some ideas: perhaps the `-` separator would work if we render human-like dates ("Jul 31st" instead of "2025-07-31")? Or changing the separator to a space so it works across integers and dates?

Reply:
Using human-like dates sounds like a good idea! However, we'll need to update the way the input widget assigns the Filter label, which might require some refactoring. I'll change the separator to a space for now, as it's simple and looks better for both integers and dates.

https://github.com/WordPress/gutenberg/blob/8a8da8af490a6fe578aba710e38bb6000a48a7b9/packages/dataviews/src/components/dataviews-filters/filter.tsx#L490-L498

Reply:
I attempted to use a space as the operator, but IMO it doesn't look quite right when dealing with integers.

<img width="180" height="36" alt="Screenshot 2025-07-29 at 13 55 37" src="https://github.com/user-attachments/assets/cc7bca08-5dde-4d07-a3bb-a38413a3a313" />

How about using a comma followed by a space for now instead?

<img width="249" height="36" alt="Screenshot 2025-07-29 at 13 56 39" src="https://github.com/user-attachments/assets/bfedb132-679f-4b3d-8023-a1b0e09958e2" />
<img width="190" height="34" alt="Screenshot 2025-07-29 at 14 00 53" src="https://github.com/user-attachments/assets/e73dec4a-05ab-4868-8042-5a053cd06835" />

Reply:
>  perhaps the - separator would work if we render human-like dates ("Jul 31st" instead of "2025-07-31")? 

Upon closer examination, perhaps we could use `field.render` for this. 🤔